pulp_smash/constants.py

```diff
@@ -12,6 +12,13 @@
     http://pulp.readthedocs.io/en/latest/dev-guide/conventions/sync-v-async.html#call-report
 """
 
+CONTENT_SOURCE_ID = 'content-world'
```

Comment:
Content sources may be named anything at all. During run-time, we could make a content source with an ID of `foo`, or of `0b935025-573d-42df-b8ff-19e0686c6d52`, or of anything else. Given that, is there a reason to define a single global content source ID?